metadata-ingestion/src/datahub/ingestion/source/tableau_common.py

```diff
@@ -537,12 +542,12 @@ def get_fully_qualified_table_name(
     platform: str,
     upstream_db: str,
     schema: str,
-    full_name: str,
+    table_name: str,
```

Comment:
This was never passed full_name...:
```
            table_urn = make_table_urn(
                self.config.env,
                upstream_db,
                table.get(tableau_constant.CONNECTION_TYPE) or "",
                schema,
                table_name,
                self.config.platform_instance_map,
                self.config.lineage_overrides,
            )
	```
5th argument is clearly `table_name`, not `full_name`